src/test/kotlin/com/zepben/evolve/cim/iec61970/base/core/EquipmentContainerTest.kt

```diff
@@ -40,4 +40,34 @@ internal class EquipmentContainerTest {
             EquipmentContainer::clearEquipment
         )
     }
+
+    @Test
+    internal fun normalFeeders() {
+        val fdr1 = Feeder()
+        val fdr2 = Feeder()
+        val fdr3 = Feeder()
+
+        val eq1 = object : Equipment() {}.addContainer(fdr1).addContainer(fdr2)
+        val eq2 = object : Equipment() {}.addContainer(fdr2).addContainer(fdr3)
```

Comment:
Can you please add some non feeder containers to the list to ensure they are being filtered out correctly. What would you expect if there are LV and HV feeders assigned?

Reply:
Currently, only HV/MV feeders. I can make it return a `Set<EquipmentContainers>` of `Feeder`s and `LvFeeder`s instead. I'm not too familiar on how these convenience functions are used outside of tests.

Reply:
This comment was intended to make you think about what it was doing rather than instruction on what needs to be changes.

It should only be returning the `Feeder` objects, not the `LvFeeder` as they are a different container type. This I was hoping would make you consider how you would get the `LvFeeder` objects, and decide it probably needs to have the equivalent functions for `lvFeeders` and `currentLvFeeders`